crates/sui-graphql-rpc/src/types/event.rs

```diff
@@ -143,13 +143,13 @@ impl Event {
     /// checkpoint sequence numbers as the cursor to determine the correct page of results. The
     /// query can optionally be further `filter`-ed by the `EventFilter`.
     ///
-    /// The `checkpoint_viewed_at` parameter is represents the checkpoint sequence number at which
-    /// this page was queried for. Each entity returned in the connection will inherit this
-    /// checkpoint, so that when viewing that entity's state, it will be from the reference of this
-    /// checkpoint_viewed_at parameter.
+    /// The `checkpoint_viewed_at` parameter represents the checkpoint sequence number at which this
+    /// page was queried. Each entity returned in the connection will inherit this checkpoint, so
+    /// that when viewing that entity's state, it will be as if it is being viewed at this
```

Comment:
```suggestion
    /// that when viewing that entity's state, it's as if it's being viewed at this
```
That's alotta little words. I think removing future tense and using contractions helps flow.